src/types.mligo

```diff
@@ -104,6 +104,7 @@ type voter =
   ; vote_type : vote_type
   }
 
+type voting_period = nat
```

Comment:
We probably on longer need this. I've removed this in #219, and uses `seconds` instead which gives better clarification.

Reply:
I see. I thought it got removed by mistake.

But isn't 'voting_period' more specific than 'seconds', thus carry more information than 'seconds'?

Reply:
How I think about this is that, `voting_period` is measured in second. 

It was not obvious previously. The type `nat` or `type voting_period = nat` does not tell that this `nat` stands for second. It could be confused for `hour` or `millisecond` and so on.

Reply:
Ah, I see. Then it could be resolved by aliasing `voting_period` to be `seconds`.

Reply:
Either ways, it's fine for me.

Reply:
I think having both may be confusing and possibly not that helpful, ideally we should have a `newtype` like in Haskell.

AFAIK ligo doesn't really have that, but it is possible to use records with a single field, I suggest:
```ocaml
type voting_period = { length : seconds }
```

This makes the usage a bit more cumbersome (you need to do `voting_period.length` all the time), but it makes it easier to see when we are dealing with just seconds/length of time VS voting periods.
Note: AFAIK this is compiled without additional costs.

Reply:
@sras pinging in case you missed the comment above

Reply:
@pasqu4le Yes, makes sense, since it provides a little more safety than using just an alias. Have updated.